entrypoint.sh

```diff
@@ -40,7 +40,7 @@ exec cloud-api-adaptor-aws aws \
 }
 
 azure() {
-test_vars AZURE_CLIENT_ID AZURE_CLIENT_SECRET AZURE_TENANT_ID AZURE_VNET_NAME AZURE_SUBNET_NAME AZURE_NSG_NAME
+test_vars AZURE_CLIENT_ID AZURE_CLIENT_SECRET AZURE_TENANT_ID AZURE_SUBSCRIPTION_ID AZURE_SUBNET_ID AZURE_IMAGE_ID
```

Comment:
You're missing AZURE_NSG_ID here

Reply:
We can create azure vm without  NSG ID as well. I think we can keep it as optional param here.